kernprof.py

```diff
@@ -204,14 +204,19 @@ def positive_float(value):
     parser.add_argument('-v', '--view', action='store_true',
                         help='View the results of the profile in addition to saving it')
     parser.add_argument('-u', '--unit', default='1e-6', type=positive_float,
-
                         help='Output unit (in seconds) in which the timing info is '
                         'displayed (default: 1e-6)')
     parser.add_argument('-z', '--skip-zero', action='store_true',
                         help="Hide functions which have not been called")
     parser.add_argument('-i', '--output-interval', type=int, default=0, const=0, nargs='?',
                         help="Enables outputting of cumulative profiling results to file every n seconds. Uses the threading module."
                         "Minimum value is 1 (second). Defaults to disabled.")
+    parser.add_argument('-p', '--prof-mod', type=str, default='',
+                        help="list of modules, functions and/or classes to profile specified by their name or path,"
```

Comment:
It's be nice if, for consistency, the help strings were capitalized similarly to the previous help strings.

Reply:
done